src/metabase/events/audit_log.clj

```diff
@@ -116,12 +118,15 @@
      :recipients (map :recipients channels)}))
 
 (methodical/defmethod events/publish-event! ::pulse-event
-  [topic {:keys [id details] :as object}]
+  [topic {:keys [id user details] :as object}]
   ;; Check if topic is a pulse or not (can be an unsubscribe event, which only contains email)
-  (let [details-map (if (some? id)
+  (let [user-id     (if (some? user)
```

Comment:
Since user-id should be the creator of the pulse in this case, add an if statement that lets us choose the creator of the pulse